seeds/breadboard-cli-tools/src/commands/lib/utils.ts

```diff
@@ -15,16 +16,22 @@ export const loadBoardFromModule = async (file: string) => {
   if (board instanceof Board == false) throw new Error(`Board ${file} does not have a default export of type Board`);
 
   return board;
-}; 
+};
 
 export const resolveFilePath = (file: string) => {
-  return path.resolve(process.cwd(), path.join(path.dirname(file), path.basename(file)));
+  return path.resolve(
+    process.cwd(),
+    path.join(path.dirname(file), path.basename(file))
+  );
 };
 
 export const loadBoard = async (file: string) => {
-  const board = await Board.load(file);
+  const boardFile: GraphDescriptor = JSON.parse(
```

Comment:
@PaulKinlan, updated this to use `Board.fromGraphDescriptor` as @dglazkov suggested